packages/commonwealth/client/scripts/views/pages/AdminPanel/RPCEndpointTask.tsx

```diff
@@ -68,6 +72,15 @@ const RPCEndpointTask = () => {
     return [];
   };
 
+  const ethChainIdEndpointValidationFn = (value: string): [ValidationStatus, string] | [] => {
+    if (value === '' || parseInt(value)) {
```

Comment:
I suggest we use regex `/[^1-9]/g.test(parseInt(value))` -- it will also check if value is 0 or it contains any thing other than a number (like parseInt("2a1") gives "2")